packages/wavs/src/subsystems/trigger.rs

```diff
@@ -685,6 +739,126 @@ impl TriggerManager {
                         }
                     }
                 }
+                StreamTriggers::AtProto { event } => {
+                    // Convert CommitAction to AtProtoAction
+                    let action_enum = match event.action {
+                        streams::atproto_jetstream::CommitAction::Create => AtProtoAction::Create,
```

Comment:
Why not just get rid of `CommitAction` and use `AtProtoAction` everywhere?

Reply:
yep! included here 01513311d28be079c08f1f97f2164244d47ed7f5